src/WebJobs.Extensions.DurableTask/Options/TraceOptions.cs

```diff
@@ -26,6 +26,19 @@ public class TraceOptions
         /// </value>
         public bool TraceInputsAndOutputs { get; set; }
 
+        /// <summary>
+        /// Gets or sets if Azure linux telemetry should include verbose logs.
+        /// </summary>
+        /// <remarks>
+        /// The default behaviour is false, which disables verbose logs. When set
+        /// to true, performance may be affected due to the amount of verbose logs.
+        /// We recommend setting this to true primarily for debugging purposes.
+        /// <value>
+        /// <c>true</c> to enable verbose telemetry; <c>false</c> otherwise.
+        /// </value>
+        /// </remarks>
+        public bool AllowVerboseLinuxTelemetry { get; set; }
```

Comment:
nit: This could just be me, but I generally like to explicitly default this to `false`, just so it is a bit more apparent what the default value is by looking. 

Some people would probably disagree with me about that, as default values for `bool` in .NET should be common knowledge.

Reply:
I agree with you. I just didn't want to mess with the format of this too much. I'll get on it

Reply:
Done!